apps/vaults/hooks/useSolverPortals.ts

```diff
@@ -347,3 +354,26 @@ export function useSolverPortals(): TSolverContext {
 	}), [expectedOut, init, onApprove, onExecute, onRetrieveAllowance]);
 }
 
+function isValidErrorObject(error: TPortalsError | unknown): error is TPortalsError {
```

Comment:
@Majorfi I've created this type guard, as I did not find an error type for an object with these props `error.response.data.message`

What kind of error is this?

Reply:
^ @Majorfi